### ISSUE TITLE
Feature Request: Add rule for reg.present to use integers for REG_DWORD

### ISSUE THREAD

**Issue by flassman** [warpnet/salt-lint#110]
reg.present can take strings as input for REG_DWORD values, however the state then does not detect if the value is already set correctly, see https://github.com/saltstack/salt/issues/55284
```
HKEY_LOCAL_MACHINE:
  reg.present:
    - vname: MyTest
    - vtype: REG_DWORD
    - vdata: '00000001'
```

so to be in the safe side, the REG_DWORD should be used with integers in vdata like this:
```
HKEY_LOCAL_MACHINE:
  reg.present:
    - vname: MyTest
    - vtype: REG_DWORD
    - vdata: 1
```

**Comment by roaldnefs** [comment]
Thanks @flassman for your feature request!

**Comment by jbouter** [comment]
@flassman Thanks for your feature request. As I am lacking the windows knowledge, what range values can `vdata` be?

**Comment by flassman** [comment]
According to https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry-value-types
the REG_DWORD is supposed to be a 32 bit integer


Am 15. November 2019 09:24:30 MEZ schrieb Jeffrey Bouter <notifications@github.com>:
>@flassman Thanks for your feature request. As I am lacking the windows
>knowledge, what range values can `vdata` be?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/warpnet/salt-lint/issues/110#issuecomment-554261717

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by roaldnefs** [comment]
This has been fixed been on the SaltStack side in https://github.com/saltstack/salt/pull/56160
